validation/validate.js

```diff
@@ -0,0 +1,43 @@
+const Ajv = require('ajv');
+const path = require('path');
+const schemaMap = require('./schemaMap.json')
```

Comment:
Does using ES6 style imports work with this version of node? If it does, please use that instead of the commonjs require.